plugins/balloontoolbar/plugin.js

```diff
@@ -723,6 +729,24 @@
 
 				CKEDITOR.ui.balloonPanel.prototype.attach.call( this, element, options );
 			};
+
+			/**
+			 * Method deregister all focusable items from balloontoolbar.
+			 *
+			 * @private
+			 * @since 4.8.1
+			 * @member CKEDITOR.ui.balloonToolbarView
+			 */
+			CKEDITOR.ui.balloonToolbarView.prototype.deregisterFocusableItems = function() {
+				var focusables = this.focusables;
+				for ( var id in focusables ) {
```

Comment:
I have to iterate here over focusables to get actually registered elements in focus manager. It appears that reference to elements is kept in `this.panels.content` even when those elements are removed from DOM. This cause an error in test where additional clear was called as `tearDown`, because another attempt to deregister focusable with those same elements have happen.